tiledb/sm/query/query.cc

```diff
@@ -513,7 +527,8 @@ Status Query::get_offsets_buffer(
         Status_QueryError("Cannot get buffer; Coordinates are not var-sized"));
   }
   if (array_schema_->attribute(name) == nullptr &&
-      array_schema_->dimension_ptr(name) == nullptr) {
+      array_schema_->dimension_ptr(name) == nullptr &&
+      !array_schema_->is_dim_label(name)) {
     return logger_->status(Status_QueryError(
```

Comment:
Consider reviewing error message to match others that we might have for dimension labels. Same on line 577.

Reply:
Done